src/main/java/core/basesyntax/strategy/handlers/BalanceOperationHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.handlers;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        Fruit fruit = transaction.getFruit();
+        if (!Storage.storage.containsKey(fruit)) {
+            Storage.storage.put(fruit, transaction.getQuantity());
+        } else {
+            Integer currentQuantity = Storage.storage.get(fruit);
+            Storage.storage.put(fruit, currentQuantity + transaction.getQuantity());
```

Comment:
I suggest that you do all database management somewhere in the "Storage" package.

Reply:
I would like to leave the logic of operations in handlers